### PR TITLE
update LI & LO types status

### PR DESCRIPTION
**Pull request by truonghongtrieu**

### CODE REVIEW

File: lo/LiTypes.php [hunk 1]
```diff
@@ -6,32 +6,51 @@
...
     const IFRAME      = 'iframe';
-    const LINK        = 'link';
-    const INTERACTIVE = 'interactive';
-    const QUESTION    = 'question';
-    const QUIZ        = 'quiz';
+
+    /**
+     * @deprecated
+     */
+    const MANUAL      = 'manual';
```
Comment (mrsinguyen) [inline lo/LiTypes.php:43]: This type is still being used. It's external lo type and we are using for Finsia

File: lo/LiTypes.php [hunk 2]
```diff
@@ -6,32 +6,51 @@
 
 class LiTypes
 {
-    const ACTIVITY    = 'activities';
-    const ATTENDANCE  = 'attendance';
     const ASSIGNMENT  = 'assignment';
+    const AUDIO       = 'audio';
     const DOCUMENT    = 'document';
+    const EVENT       = 'event';
     const H5P         = 'h5p';
```
Comment (pvmchau) [inline lo/LiTypes.php:13]: `h5p` type is deprecated also

File: lo/LiTypes.php (hunk 1 above)
Comment (truonghongtrieu) [reply lo/LiTypes.php]: thanks @mrsinguyen, updated

File: lo/LiTypes.php (hunk 2 above)
Comment (truonghongtrieu) [reply lo/LiTypes.php]: thanks @pvmchau , updated

**Review by pvmchau — APPROVED** [review]

**Review by mrsinguyen — APPROVED** [review]

**Pull request merged**